examples/OECC/setup.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
```

Comment:
A comment to reflect a summary or purpose of the script would be welcomed. Same for all the .sh scripts.